packages/core/src/html/CustomListIconProcessor.ts

```diff
@@ -18,10 +18,12 @@ interface IconAttributes {
   width?: string;
   height?: string;
   spacing?: string;
+  oneOff?: boolean;
```

Comment:
This doesn't need to be undefined
```suggestion
  oneOff: boolean;
```

Reply:
It prevents the IconAttributes object from being undefined as a whole